style.css

```diff
@@ -264,3 +267,8 @@ h4 {
 .btn:disabled {
   color: #7d73fc;
 }
```

Comment:
For your buttons kindly set width: 89px and height: 48px like in the figma template. Please inspect the template carefully.